mongo/integration/unified/collection_operation_execution.go

```diff
@@ -830,6 +830,36 @@ func executeListIndexes(ctx context.Context, operation *operation) (*operationRe
 	return newCursorResult(docs), nil
 }
 
+func executeRenameCollection(ctx context.Context, operation *operation) (*operationResult, error) {
+	coll, err := entities(ctx).collection(operation.Object)
+	if err != nil {
+		return nil, err
+	}
+
+	var toName string
+	elems, _ := operation.Arguments.Elements()
+	for _, elem := range elems {
+		key := elem.Key()
+		val := elem.Value()
+
+		switch key {
+		case "to":
+			toName = val.StringValue()
+		default:
+			return nil, fmt.Errorf("unrecognized rename option %q", key)
+		}
+	}
+
```

Comment:
```suggestion
    if toName == "" {
        return nil, newMissingArgumentError("to")
    }
```
Should this be a required field or will runCommand just do nothing if it's missing?  If it _does_ just do nothing, do we want to allow that?

Reply:
Ah it should be required! I'm not sure of the behavior of `renameCollection` if it's missing; I think it might rename the collection to the empty string or throw a server error.